.editorconfig

```diff
@@ -0,0 +1,75 @@
+# EditorConfig: https://editorconfig.org/
```

Comment:
Should we really be providing opinionated defaults for the user's own repo?

This is a genuine question. I could see the benefits either way.

Reply:
removed (including checkpatch stuff)